### PR TITLE
Bom Freq on Part Record

### PR DESCRIPTION
**Pull request by RossStewart**

### CODE REVIEW

File: src/Persistence.LinnApps/ServiceDbContext.cs [hunk 1]
```diff
@@ -465,9 +465,9 @@ private void BuildParts(ModelBuilder builder)
             e.Property(p => p.SingleSourcePart).HasColumnName("SINGLE_SOURCE_PART").HasMaxLength(1);
             e.Property(p => p.SafetyCertificateExpirationDate).HasColumnName("SAFETY_CERTIFICATE_EXPIRY_DATE");
             e.Property(p => p.SafetyDataDirectory).HasColumnName("SAFETY_DATA_DIRECTORY").HasMaxLength(500);
-            e.Property(p => p.LinnProduced).HasColumnName("LINN_PRODUCED")
-                .HasMaxLength(1).HasColumnType("VARCHAR2").IsUnicode(false);
-            e.Property(p => p.BomType).HasColumnName("BOM_TYPE").HasMaxLength(1).HasColumnType("VARCHAR2").IsUnicode(false);
+            e.Property(p => p.LinnProduced).HasColumnName("LINN_PRODUCED").HasMaxLength(1);
+            e.Property(p => p.BomType).HasColumnName("BOM_TYPE").HasMaxLength(1);
+            e.Property(p => p.BomVerifyFreqWeeks).HasColumnName("BOM_VERIFY_FREQ_WEEKS").HasMaxLength(4);
```
Comment (lewisrenfrew) [inline src/Persistence.LinnApps/ServiceDbContext.cs:470]: I take it those .IsUnicode(false) calls you deleted were redundant? I don't remember why they were there in the first place

**Review by lewisrenfrew — APPROVED** [review]

File: src/Persistence.LinnApps/ServiceDbContext.cs (hunk 1 above)
Comment (RossStewart) [reply src/Persistence.LinnApps/ServiceDbContext.cs:470]: Will be resubmitting this with my changes in.  Had a bit of a branch issue but I'll explain it in stand up.

Comment (lewisrenfrew) [reply src/Persistence.LinnApps/ServiceDbContext.cs:470]: what's the issue? All looks ok to me apart from the test fail which is an easy fix

Comment (lewisrenfrew) [reply src/Persistence.LinnApps/ServiceDbContext.cs:470]: Why does that matter? you merged master into this branch anyway and the diff looks sensible - I would have just fixed the test here and merged this branch (unless I'm missing something)

Comment (richardiphillips) [reply src/Persistence.LinnApps/ServiceDbContext.cs:470]: was it telling you that it couldn't be merged? Seems like all you need to do is fix the test like Lewis says

Comment (RossStewart) [reply src/Persistence.LinnApps/ServiceDbContext.cs:470]: Reverted the removal of the Unicode (false) calls too. Wasn't intentional. That was partly why I wasn't convinced I was totally up to date there but it's fine.

**Pull request merged**